### PR TITLE
Slim dashboard package size

### PR DESCRIPTION
**Pull request by erikphansen**
## Description
Dashboard package sizes were shrunk from ~2.4MB total to ~1.2MB total.

This was done by no longer importing all form configs into the Dashboard helpers. This was _only_ done because the unreleased (and since abandoned) 686 form was blocked from users who did not have a 30% disability rating. This PR also removed the old 686 form code and all code related to checking if a user was "authorized" to see or have access to a particular form. That authorization checking logic was never actually used and was actually pretty tightly bound to the old 686. I spoke with @micahchiang about deleting all of that code and he was on board.

## Testing done
Local + existing tests

## Screenshots
N/A

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by erikphansen** [comment]
@ncksllvn I fixed this! It took less than 25 months!

**Review by Mottie — APPROVED** [review]
2020-11-13 07_04_02-Slim dashboard package size by erikphansen · Pull Request #14950 · department-of
ghe-emoji

**Review by micahchiang — APPROVED** [review]
👀

**Review by cvalarida — APPROVED** [review]
This is amazing. Good work! :raised_hands:

File: src/applications/personalization/dashboard/components/FormItem.jsx [hunk 1]
```diff
@@ -89,15 +84,7 @@ class FormItem extends React.Component {
       </DashboardAlert>
     );
 
-    const content = isExpired ? expiredView : activeView;
-    if (isAuthorizable) {
-      return (
-        <AuthorizationComponent formConfig={formConfig} isVisible>
```
Comment (erikphansen) [inline src/applications/personalization/dashboard/components/FormItem.jsx:95]: This was the only place the `AuthorizationComponent` was used and also the only reason all of the form configs were being loaded into the helper file in the first place. And the old, unused 686 was the only form that passed the `isAuthorizable()` test.

**Comment by erikphansen** [comment]
@cvalarida It was your talk about decoupling the form system from the Dashboard code that made me realize why the Dashboard bundle was so large 🙄

**Review by sanlouise — APPROVED** [review]
Half the package size, impressive! Great cleanup! 👍

**Review by Clark-W-Griswold — APPROVED** [review]
👍

**Pull request merged**